pom.xml

```diff
@@ -1074,6 +1074,7 @@
                 <module>tensorflow-java</module>
                 <module>xstream</module>
                 <module>webrtc</module>
+				<module>persistence-modules/java-mongodb</module> <!-- long running -->
```

Comment:
no need to include this in a submodule fashion. persistence-modules is already present in the root pom

Reply:
persistence-modules is in the jdk 8 profile. I migrate this project to run under jdk9 and above profile. The reason why I migrated to the jdk9 was because morphia-core requires jdk 11, that is why added in the sub-module fashion just that module and I removed the module from the pom.xml the java-mongodb.

Reply:
Ok